topics/02_Controls/igRadialGauge/02_Configuring/04_igRadialGauge_Configuring_Labels.md

```diff
@@ -129,8 +137,6 @@ The following samples provide additional information related to this topic:
 
 - [Gauge Needle](%%SamplesUrl%%/radial-gauge/gauge-needle): Displayed as a pointer, the Needle indicates a single value on a scale. The options pane below allows you to interact with the Radial Gauge control’s Needle.
 
-- [Label Settings](%%SamplesUrl%%/radial-gauge/label-settings): This sample demonstrates how to configure the Radial Gauge control’s Label settings. Use the slider to see how the `labelInterval` and `labelExtent` properties affect the Label.
```

Comment:
There are references to the [Label Settings] (%%SamplesUrl%%/radial-gauge/label-settings) sample in other radial gauge topics. All those references should be updated to point to the newly added embed sample. Something like (igRadialGauge-Configuring-Labels.html#lable-example)